idf_component_examples/Hello_world/README.md

```diff
@@ -0,0 +1,63 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C6 | ESP32-H2 | ESP32-S2 | ESP32-S3 |
+| ----------------- | ----- | -------- | -------- | -------- | -------- | -------- | -------- |
+
+# _Hello world example_
+
+This is the simplest buildable example made to be used as a template for new projects running Arduino-esp32 as an ESP-IDF component.
+See [Arduino-esp32](https://components.espressif.com/components/espressif/arduino-esp32) in ESP Registry.
+
+## How to use example
+
+To create a ESP-IDF project from this example with the latest relase of Arduino-esp32, you can simply run command: `idf.py create-project-from-example "espressif/arduino-esp32:hello_world"`.
+ESP-IDF will download all dependencies needed from the component registry and setup the project for you.
+
+If you want to use cloned Arduino-esp32 repository, you can build this example directly.
+Go to the example folder `arduino-esp32/idf_component_examples/Hello_world`.
+First you need to comment line 6 `pre_release: true` in examples `/main/idf_component.yml`.
```

Comment:
I just tested it in a separated folder.
It is also necessary to comment out the line 5 `override_path: '../../../'` as it may not exist.

Reply:
Actually, it should be removed by the ESP Registry so if you get it from there as an example, this won't exist. On other components it is done same way, for development purposes. If I understood correctly :)